src/Data/ProgrammesDb/EntityRepository/VersionRepository.php

```diff
@@ -140,6 +140,30 @@ public function findAvailableByProgrammeItem(string $programmeDbId): array
         return $qb->getQuery()->getResult(Query::HYDRATE_ARRAY);
     }
 
+    public function findLinkedVersionsForProgrammeItem(string $programmeItemDbId): array
```

Comment:
If this is for all versions, should we call it `all` instead of `linked`? "Linked" sounds special.

Reply:
It's not all versions, it's the versions Faucet chooses as streamable, canonical and downloadable

Reply:
Could we have VERY clear docblocks on all of these methods in that case?